tests/yam/agama/agama_auto.pm

```diff
@@ -24,6 +32,7 @@ sub run {
 
     my @tags = ("welcome-to", "login");
     assert_screen(\@tags, 960);
+    reset_consoles;
```

Comment:
Yes, without this it will be failed to select root console in valid_product, https://openqa.opensuse.org/tests/4492078#step/validate_product/4. I think it may for we already login root console but haven't logout.

Reply:
the should be similar after reboot both interactive and auto, perhaps we just need to schedule grub_test and first_boot? Otherwise I don't get why this fail does not occur in interactive...

Reply:
In fact, I found in wait_boot_textmode it reset_consoles also.  So do we need to make the auto to be same with interactive?  We have reason to create own needle for grub2 in auto:

# For agama test, it is too short time to match the grub2, so we create
    # a new needle to avoid too much needles loaded.
    assert_screen('grub2-agama', 120);

Reply:
Done to add grub_test, first_boot for auto test. Updated VRs also.